ecommerce/views.py

```diff
@@ -16,3 +17,14 @@ class SignUpView(generic.CreateView):
     form_class = UserCreationForm
     success_url = reverse_lazy('login')
     template_name = 'registration/signup.html'
+
+def userpage(request):
```

Comment:
đặt tên phân cách nhau _
```suggestion
def user_profile(request):
```